### PR TITLE
Update to FileExplorer documentation

### PR DESCRIPTION
**Pull request by ryestew**

### CODE REVIEW

File: docs/file_explorer.md [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
+-----------------------------
+Right click on a file or a folder and the **context menu** will appear. 
 
-Enables you to copy files from the browser storage to another instance
-(URL) of Remix.
+![](images/a-file-ex-rt-click.png)
 
-Connect your filesystem to Remix
---------------------
+You can rename or a file or a folder.  You can also delete a file or a folder. These functions also work with folders and files on your hard drive when using RemixD.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:51]: `You can rename or a file or a folder` => `You can rename a file or a folder`

Also I think, both lines can be combined as : `You can rename and delete a file or a folder`

File: docs/file_explorer.md [hunk 2]
```diff
@@ -1,54 +1,53 @@
...
-Publishes all files from the browser folder to a gist.
-Gist API has changed in 2018 and it unfortunately requires users to be authenticated to be able to publish a gist.
+The icon marked **C.** above. Publishes all files from the browser folder to a gist.  Only file in the root of **browser** will be published.  Files in subfolders will not be publish to the Gist.
+Gist API has changed in 2018 and **requires** users to be authenticated to be able to publish a gist.
+
+Click [this link](https://github.com/settings/tokens) to Github tokens setup and select Generate new token. Then check the **Create gists** checkbox and generate a new token.
 
-Click [this link](https://github.com/settings/tokens) to Github tokens setup  and select Generate new token.
-Then check only Create gists checkbox and generate a new token.
+Take the token and paste it in Remix's **Setting** module in the **Github Access Token** section. And then click Save. Now you should be able to use the feature.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:38]: `Setting` => `Settings`

Comment (ryestew) [reply docs/file_explorer.md]: done

File: docs/file_explorer.md (hunk 1 above)
Comment (ryestew) [reply docs/file_explorer.md]: Updated

**Pull request merged**